disnake/channel.py

```diff
@@ -930,7 +930,7 @@ def permissions_for(self, obj: Union[Member, Role], /) -> Permissions:
         return base
 
 
-class VoiceChannel(VocalGuildChannel):
+class VoiceChannel(disnake.abc.Messageable, VocalGuildChannel):
```

Comment:
I think Messageable is defined in this channel, so use ``class VoiceChannel(Messageable, VocalGuildChannel):``

Reply:
The `Messageable` class is not imported; only the `disnake.abc` module. Importing explicitly `disnake.abc.Messageable` follows the import format used by other classes in this module.

Reply:
ah, I see it was already a thing here, gotcha.